spec/models/activity_spec.rb

```diff
@@ -736,14 +736,6 @@
       it { should validate_presence_of(:extending_organisation_id).on(:update_extending_organisation) }
     end
 
-    context "when the form state is blank" do
-      it "allows updates to be made to other fields set on creation" do
-        blank_activity = create(:activity, funding_organisation_name: "old", form_state: :blank)
```

Comment:
Was originally wondering if test should be kept in, and another field substituted for `funding_organisation_name` in this example, but since the `:blank` form state is only here for legacy reasons, it seems fine to remove this 👍